Content.Server/Fluids/EntitySystems/DrainSystem.cs

```diff
@@ -1,18 +1,107 @@
+using System.Linq;
 using Content.Server.Chemistry.Components.SolutionManager;
 using Content.Server.Fluids.Components;
 using Content.Server.Chemistry.EntitySystems;
+using Content.Server.DoAfter;
+using Content.Server.Popups;
 using Content.Shared.FixedPoint;
 using Content.Shared.Audio;
+using Content.Shared.Database;
+using Content.Shared.DoAfter;
+using Content.Shared.Examine;
+using Content.Shared.Fluids;
+using Content.Shared.Interaction;
+using Content.Shared.Tag;
+using Content.Shared.Verbs;
 using Content.Shared.Fluids.Components;
 using Robust.Shared.Collections;
+using Robust.Shared.Random;
+using Robust.Shared.Utility;
 
 namespace Content.Server.Fluids.EntitySystems
```

Comment:
wont require you to but you should change this to a file scoped namespace since you are already tweaking this file so much

Reply:
Didn't know about file scoped namespaces, learnt something today